src/index.js

```diff
@@ -22,15 +25,19 @@ exports.codes = Object.freeze(codes)
 /**
  * Convert the given multihash to a hex encoded string.
  *
- * @param {Buffer} hash
+ * @param {Uint8Array} hash
```

Comment:
`Buffer` is also a `Uint8Array`, but should this perhaps be `Uint8Array|Buffer` for more clarity? I'm not sure about this, I leave this up to you, as you've spend more time on this kind of conversion also in other libraries.